### DISCUSSION TITLE
404 Client Error: Not Found for url: https://raw.githubusercontent.com/mrdbourke/pytorch-deep-learning/main/data/pizza_steak_sushi/04-pizza-dad.jpeg

### DISCUSSION THREAD

**Discussion by Muzammalonhub** [mrdbourke/pytorch-deep-learning#1138] [Q&A]
Dear Mr Bourke, I am facing an issue during loading an custom image in the video course "pytorch_custom_dataset_video" . When I go to read an custom image, 
# Read in custom image 
custom_image_uint8 = torchvision.io.read_image(str(custom_image_path))
custom_image_uint8

the following error I am facing. 
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
[<ipython-input-85-026d7d9ac1c6>](https://localhost:8080/#) in <cell line: 4>()
      2 
      3 # Read in custom image
----> 4 custom_image_uint8 = torchvision.io.read_image(str(custom_image_path))
      5 custom_image_uint8

2 frames
[/usr/local/lib/python3.10/dist-packages/torch/_ops.py](https://localhost:8080/#) in __call__(self, *args, **kwargs)
   1114         if self._has_torchbind_op_overload and _must_dispatch_in_python(args, kwargs):
   1115             return _call_overload_packet_from_python(self, args, kwargs)
-> 1116         return self._op(*args, **(kwargs or {}))
   1117 
   1118     # TODO: use this to make a __dir__

RuntimeError: Unsupported image file. Only jpeg, png and gif are currently supported.

**Comment by gvtsch** [comment]
Hej,

Have you tried to open it manually? I can't, getting the same error: 404: Not Found
But if you look into the repo, you can find this one: https://raw.githubusercontent.com/mrdbourke/pytorch-deep-learning/main/data/04-pizza-dad.jpeg
It looks like the file has been moved to the parent folder.
